platform/linux-generic/test/wrapper-script.sh

```diff
@@ -0,0 +1,3 @@
+#wrapper script for pre setting environment for validation suit.
```

Comment:
@GBalakrishna I understand this PR will make linux-dpdk platform build really run all the tests, can you help elaborate why this script with only comments can help archive this?

Reply:
@heyi-linaro as I written it in the comment section in the script, it is to be aligned with linux-dpdk wrapper-script. I couldn't find any better way of enabling wrapper-script from linux-dpdk.